src/components/search-box.vue

```diff
@@ -1,26 +1,27 @@
 <template>
-  <div class="x-input-group x-input-group--card x-border-color--lead">
+  <div class="x-input-group x-input-group-lead x-rounded-sm">
     <SearchInput
       :autofocus="false"
       :placeholder="$t('searchBox.placeholder')"
-      :class="$x.device === 'mobile' ? 'x-padding--left-05' : 'x-padding--left-08'"
+      :instant="false"
+      class="desktop:x-pl-40"
```

Comment:
The input group's first child now has a margin of `16px`, so now the input is 16px more to the right than it should be in desktop (in mobile is ok).

```suggestion
      class="desktop:x-pl-24"
```

Reply:
fixed